sdk/src/client.ts

```diff
@@ -100,9 +99,12 @@ export class VaultClient {
         strategyType: StrategyType,
         rebalanceMode: RebalanceMode,
         owner: PublicKey,
-        feeData: FeeArgs,
-        poolSizeLimit: number = 10000000000,
+        referralFeeOwner: PublicKey,
+        feeCarryBps: number = 0,
+        feeMgmtBps: number = 0,
+        referralFeePct: number = 0,
         allocationCapPct: number = 100,
+        depositCap?: number,
```

Comment:
I think it's more readable and consistent to pass `VaultConfig` type as argument. You did that for the `updateConfig` method